packages/react-sdk/src/components/BackgroundFilters/BackgroundFilters.tsx

```diff
@@ -217,6 +218,10 @@ export const BackgroundFiltersProvider = (
     call?.tracer,
   ]);
 
+  const setLoading = useCallback((loading: boolean) => {
+    setIsLoading(loading);
+  }, []);
```

Comment:
No need to wrap `setIsLoading` in another `useCallback`. The reference is guaranteed to be stable